src/GovUk.Education.ExploreEducationStatistics.Data.Api/Controllers/TableBuilderController.cs

```diff
@@ -94,22 +94,23 @@ public class TableBuilderController : ControllerBase
             }
 
             return await _tableBuilderService
-                .Query(releaseVersionId, request.AsFullTableQuery(), cancellationToken)
+                .Query(releaseVersionId, request.AsFullTableQuery(), request.BoundaryLevel, cancellationToken)
```

Comment:
Does any request hit this endpoint with a `BoundaryLevel` value set. The intention here is that we get rid of `BoundaryLevel` from `FullTableQueryRequest` and hopefully nothing uses this and the value can be `null`, otherwise we should turn it into a query parameter like the other endpoints.